packages/paste-website/stories/PageTemplates.stories.tsx

```diff
@@ -860,3 +880,505 @@ export const SettingsOverviewExample = (): JSX.Element => {
+                >
+                  2
+                </Text>
+              </Box>
+              <Box width="100%">
+                <FormSection>
+                  <FormSectionHeading variant="heading30">Name your schema</FormSectionHeading>
+                  <FormSectionDescription>
+                    Pick a name to help you identify this space in the warehouse, or use the default name provided. This
+                    can’t be changed once the warehouse is connected.
+                  </FormSectionDescription>
+                  <FormControl>
+                    <Label htmlFor={input1} required>
+                      Name
+                    </Label>
+                    <Input type="text" id={input1} name="input1" required value="vik_test" />
+                  </FormControl>
+                </FormSection>
+              </Box>
+            </Box>
+            <Box display="flex">
+              <Box width="size10">
+                <Text
+                  as="span"
+                  color="colorTextWeak"
+                  fontWeight="fontWeightSemibold"
+                  fontSize="fontSize60"
+                  lineHeight="lineHeight60"
+                >
+                  3
+                </Text>
+              </Box>
+              <Box width="100%">
+                <FormSection>
+                  <FormSectionHeading variant="heading30">Enter your credentials</FormSectionHeading>
+                  <FormSectionDescription>Copy and paste these credentials from AzureSQL.</FormSectionDescription>
+                  <FormControl>
+                    <Label htmlFor={input2}>Server name</Label>
+                    <Input type="text" id={input2} name="input2" />
+                  </FormControl>
+                  <FormControl>
+                    <Label htmlFor={input3}>Database</Label>
+                    <Input type="text" id={input3} name="input3" />
+                  </FormControl>
+                  <FormControl>
+                    <Label htmlFor={input4}>User</Label>
+                    <Input type="text" id={input4} name="input4" />
+                  </FormControl>
+                  <FormControl>
+                    <Label htmlFor={input5}>Password</Label>
+                    <Input type="text" id={input5} name="input5" />
+                  </FormControl>
+                </FormSection>
+              </Box>
+            </Box>
+            <Box display="flex">
+              <Box width="size10">
+                <Text
+                  as="span"
+                  color="colorTextWeak"
+                  fontWeight="fontWeightSemibold"
+                  fontSize="fontSize60"
+                  lineHeight="lineHeight60"
+                >
+                  4
+                </Text>
+              </Box>
+              <Box width="100%">
+                <FormSection>
+                  <FormSectionHeading variant="heading30">Test connection</FormSectionHeading>
```

Comment:
todo: remove bottom margin from heading

Reply:
we haven't added the marginBottom prop to FormSectionHeading. will do

Reply:
update from standup: don't add prop, remove top margin instead